x-pack/platform/packages/shared/kbn-apm-types/src/es_schemas/raw/fields/span_links.ts

```diff
@@ -9,3 +9,8 @@ export interface SpanLink {
   trace: { id: string };
   span: { id: string };
 }
+
+export interface RawOtelSpanLink {
```

Comment:
Should we drop the `Raw` from the name?

```suggestion
export interface OtelSpanLink {
```

Reply:
yeah. that's not needed. thanks